### PR TITLE
add all controller files

### PR DESCRIPTION
**Pull request by REFLAXua**
look at this

### CODE REVIEW

File: controllerReports.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    const query = 'SELECT * FROM MentionReport';
+    const [results] = await db.query(query);
+    res.json(results);
+  } catch (err) {
+    console.error(err);
+    res.status(500).json({ error: 'Internal Server Error' });
+  }
+};
+
+const getMentionReportById = async (req, res) => {
```
Comment (REFLAXua) [inline controllerReports.js:14]: good looking variable

File: controllerReview.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    const query = 'SELECT * FROM PubReview';
+    const [results] = await db.query(query);
+    res.json(results);
+  } catch (err) {
+    console.error(err);
+    res.status(500).json({ error: 'Internal Server Error' });
+  }
+};
+
+const getPubReviewById = async (req, res) => {
```
Comment (REFLAXua) [inline controllerReview.js:14]: another good looking variable

**Review by REFLAXua — COMMENTED** [review]
so, there's nothing to change. Just some compliments.

File: controllerReview.js (hunk 2 above)
Comment (snxwisalive) [reply controllerReview.js:14]: i think you meant function

File: controllerReports.js (hunk 1 above)
Comment (REFLAXua) [reply controllerReports.js:14]: some missunderstanding here - not a variable, but const

**Pull request merged**